test/metabase/search/appdb/scoring_test.clj

```diff
@@ -101,11 +101,11 @@
           ;; Note that, ceteris paribus, the ordering in the database is currently stable - this might change!
           ;; Due to stemming, we do not distinguish between exact matches and those that differ slightly.
         (is (= [["card" 1 "orders"]
-                ["card" 4 "order"]
-                  ;; We do not currently normalize the score based on the number of words in the vector / the coverage.
+                ;; We do not currently normalize the score based on the number of words in the vector / the coverage.
                 ["card" 5 "orders, invoices, other stuff"]
                 ["card" 6 "ordering"]
-                  ;; If the match is only in a secondary field, it is less preferred.
+                ["card" 4 "order"]
```

Comment:
It looks like exact matches get their scores implicitly boosted now?

Strictly speaking we should then adjust the weights to account for the double counting, but I guess they weren't set too precisely before this.